src/main/java/stelnet/commodity/view/DeleteCommodityIntel.java

```diff
@@ -13,7 +14,7 @@
 public class DeleteCommodityIntel extends Button {
 
     public DeleteCommodityIntel(final String commodityId) {
-        super(new Size(200, 24), "Delete This", true, Misc.getButtonTextColor());
+        super(new Size(200, 24), "commodityDeleteThis", true, Misc.getButtonTextColor());
```

Comment:
It would be more explicit to show L10n.get("commodityDeleteThis") when calling the super method. Otherwise we are hiding the fact that it will do it in the button class. When we look it from the child class perspective, we don't understand the intent of this string.

Reply:
yes, you are right... but that means a lot of places to edit instead....

Reply:
i'll fix it as i don't want to have ui tied to the rest of the mod